jasmine/repositories.bzl

```diff
@@ -14,35 +14,30 @@ _ATTRS = {
 }
 
 def _jasmine_repo_impl(repository_ctx):
-    build_content = """# Generated by @aspect_rules_jasmine//jasmine:repositories.bzl
-load("@aspect_rules_js//js:defs.bzl", "js_binary")
+    # Base BUILD file for this repository
+    repository_ctx.file("BUILD.bazel", """\
+# Generated by @aspect_rules_jasmine//jasmine:repositories.bzl
 load("@aspect_rules_jasmine//jasmine/private:{version}/defs.bzl", "npm_link_all_packages")
-load("@aspect_bazel_lib//lib:copy_file.bzl", "copy_file")
+load("@aspect_bazel_lib//lib:directory_path.bzl", "directory_path")
 
 npm_link_all_packages(name = "node_modules")
 
-copy_file(
-    name = "copy_runner",
-    src = "@aspect_rules_jasmine//jasmine/private:runner.js",
-    out = "runner.js",
-)
-
-js_binary(
-    name = "{name}",
-    data = [
-        ":node_modules/jasmine",
-        ":node_modules/jasmine-core"
-    ],
-    entry_point = ":runner.js",
+directory_path(
+    name = "jasmine_entrypoint",
+    directory = ":node_modules/jasmine/dir",
+    path = "bin/jasmine.js",
```

Comment:
did we lose all the sharding and test xml features from https://github.com/bazelbuild/rules_nodejs/blob/stable/packages/jasmine/jasmine_runner.js ?

Reply:
nope. being implemented over #7

Reply:
okay sounds good, one step at a time